src/main/java/com/purbon/kafka/topology/AccessControlManager.java

```diff
@@ -30,30 +32,23 @@
 
   private AccessControlProvider controlProvider;
   private ClusterState clusterState;
-  private Map<String, String> cliParams;
+  private Configuration config;
 
   public AccessControlManager(AccessControlProvider controlProvider) {
-    this(controlProvider, new ClusterState(), new HashMap<>());
-  }
-
-  public AccessControlManager(
-      AccessControlProvider controlProvider, Map<String, String> cliParams) {
-    this(controlProvider, new ClusterState(), cliParams);
+    this(controlProvider, new ClusterState());
   }
 
   public AccessControlManager(AccessControlProvider controlProvider, ClusterState clusterState) {
-    this(controlProvider, clusterState, new HashMap<>());
+    this(controlProvider, clusterState, new MapConfiguration(new HashMap<>()));
```

Comment:
map configuration? name sounds confusing a bit. What do you think?

Reply:
MapConfiguration is a Class provided by Config2.